src/scss/components/_skip-to-content.scss

```diff
@@ -0,0 +1,10 @@
+.c-skip-to-content {
```

Comment:
This is another thing that's not really documented, but it's included in the system.

This doesn't make sense to show as an isolated component.

It could be documented better by creating a placeholder nav pattern, putting that in a new layout, then creating a new `src/markup/pages/demos` page that shows/explains "skip to content" behavior.

Reply:
What about keeping it in the button system and then use a modifier class (`--skip-to`) to add the hide/show on focus functionality.

Also, while tabbing through the general page to test the link I noticed there isn't a focus state. What do you think about adding safe focus to the starter?

Reply:
Okay, yeah. I'll update the color to something with more contrast, then make a card about documenting this better.